modules/ossm-installing-multi-primary-multi-network-kiali.adoc

```diff
@@ -0,0 +1,216 @@
+// This procedure is used in the following assembly:
+// * install/ossm-multi-cluster-topologies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-installing-multi-primary-multi-network-kiali_{context}"]
+= Installing a multi-cluster mesh by using Kiali
+
+Install {istio} in the multi-primary multi-network topology on two {ocp-product-title} clusters.
```

Comment:
nit: we're installing Kiali here in a multi-cluster configuration on two clusters and not istio.